frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx

```diff
@@ -207,20 +208,26 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>([
             () => [(_, props) => props.allowNonCapturedEvents],
             (allowNonCapturedEvents: boolean | undefined) => allowNonCapturedEvents ?? false,
         ],
+        enablePreaggregatedTableHints: [
+            () => [(_, props) => props.enablePreaggregatedTableHints],
+            (enablePreaggregatedTableHints) => !!enablePreaggregatedTableHints,
+        ],
         taxonomicGroups: [
-            (s) => [
-                s.currentTeam,
-                s.currentProjectId,
-                s.groupAnalyticsTaxonomicGroups,
-                s.groupAnalyticsTaxonomicGroupNames,
-                s.eventNames,
-                s.schemaColumns,
-                s.metadataSource,
-                s.propertyFilters,
-                s.eventMetadataPropertyDefinitions,
-                s.eventOrdering,
-                s.maxContextOptions,
-            ],
+            (s) =>
+                [
+                    s.currentTeam,
+                    s.currentProjectId,
+                    s.groupAnalyticsTaxonomicGroups,
+                    s.groupAnalyticsTaxonomicGroupNames,
+                    s.eventNames,
+                    s.schemaColumns,
+                    s.metadataSource,
+                    s.propertyFilters,
+                    s.eventMetadataPropertyDefinitions,
+                    s.eventOrdering,
+                    s.maxContextOptions,
+                    s.enablePreaggregatedTableHints,
+                ] as any, // workaround as Kea's SelectorTuple has a limit of 11 items: https://github.com/keajs/kea/blob/v3.1.5/src/types.ts#L162-L174
```

Comment:
I've worked around these before by grouping them up and making another selector just to get a group of props

e.g. you could add a selector that returns {currentTeam, currentProjectId} and then taxonomicGroups can use that selector.

Reply:
This is a nice option. Rafa and Marius updated Kea, and now we can go up to 16 🎉